src/stddef.h

```diff
@@ -1,12 +1,20 @@
 #define U32_MAX 2147483647
 #define SIZE_T_MAX U32_MAX
 
+//this is needed for networking later
+typedef unsigned long long u128;
+typedef unsigned char* u_str;
```

Comment:
This is not needed for now and networking was never in the scope of the kernel/OS but these additions could be useful. You should really stick to the naming convention and formatting of this file when adding them though. `str` and other variations make the code to rusty and are not really needed.